test/github-scan

```diff
@@ -36,8 +36,13 @@ def main():
     opts = parser.parse_args()
 
     github = testinfra.GitHub("/repos/cockpit-project/cockpit/")
+
     for entry in github.scan(not opts.dry, opts.context, opts.except_context):
         sys.stdout.write("{0}: {1}\n".format(entry.priority, entry.task.description()))
 
+    for image, wait_time in github.scan_image_wait_times().items():
+        if wait_time > 0:
```

Comment:
Why this condition? If we want to recreate the images now (not just wait), we should show that as well. We could use "if wait_time <= 0:" and write something like "... will be refreshed"